data/transactions/logic/box.go

```diff
@@ -156,6 +156,10 @@ func opBoxDel(cx *EvalContext) error {
 	return cx.Ledger.DelBox(cx.appID, name, appAddr)
 }
 
+const boxPrefix = "bx:"
```

Comment:
introducing constants to be shared between `MakeBoxKey()` and `GetAppAndNameFromKey()`